contrib/perf/post-pr-comment.py

```
@@ -188,6 +192,111 @@ def create_comment_chunks(reports: list[dict], run_id: str, run_url: str, status
     return comments
 
 
+def find_existing_comment(owner: str, repo: str, pr_number: int, token: str) -> dict | None:
+    """Find the most recent performance test results comment on the PR."""
+    url = f"https://api.github.com/repos/{owner}/{repo}/issues/{pr_number}/comments"
+    headers = {
+        'Authorization': f'token {token}',
+        'Accept': 'application/vnd.github.v3+json'
+    }
+
+    response = requests.get(url, headers=headers)
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)